src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -130,6 +130,17 @@ async function initializeConnection(options: CircuitStartOptions, logger: Logger
     await connection.start();
   } catch (ex) {
     unhandledError(connection, ex, logger);
+
+    if (ex.innerErrors && ex.innerErrors.some(e => e.errorType === 'UnsupportedTransportError' && e.transport === 'WebSockets')) {
+      showErrorNotification('Unable to connect, please ensure you are using an updated browser that supports WebSockets.');
+    } else if (ex.innerErrors && ex.innerErrors.some(e => e.errorType === 'FailedToStartTransportError' && e.transport === 'WebSockets')) {
+      showErrorNotification('Unable to connect, please ensure WebSockets are available. A VPN or proxy may be blocking the connection.');
```

Comment:
>Unable to connect, please ensure WebSockets are available.

Unable to connect, please ensure WebSockets are available **on the server**.

Reply:
I'm not so sure about this as this message is end-user facing and hence I would like to avoid "on the server" as it has no end user configurability. Additionally, it may not be a server issue if the local client VPN/proxy is blocking the connection.